### DISCUSSION TITLE
Replication failed error,badarg

### DISCUSSION THREAD

**Discussion by csemio** [apache/couchdb#3793] [Q&A] [answered]
Hi team,
I'have a problem when i start a new replication between two existing databases, i got this message :
  "_replication_state_reason": "{error,badarg}"

We have a 3 nodes clusters on couchdb 3.0.0
We have a database with 30Millions docs
Replication doc :
`{
  "_id": "090e714507f62a025bab8d166000753d",
  "_rev": "2-14d1008fc3552d842bcffcb101e7ebfa",
  "user_ctx": {
    "name": "admin",
    "roles": [
      "_admin",
      "_reader",
      "_writer"
    ]
  },
  "source": {
    "url": "http://localhost:5984/activity-logs",
    "headers": {
      "Authorization": "Basic XXXXXXXXXXXXXXXX"
    }
  },
  "target": {
    "url": "http://localhost:5984/activity-logs-new",
    "headers": {
      "Authorization": "Basic XXXXXXXXXXXXXXXX"
    }
  },
  "create_target": false,
  "continuous": false,
  "owner": "admin",
  "_replication_state": "failed",
  "_replication_state_time": "2021-10-15T15:46:02Z",
  "_replication_state_reason": "{error,badarg}"
}`

many thanks.

**Comment by nickva** [comment, marked as answer]
See if there are any log lines with then `badarg` term in them. Then capture a few lines around that that may have stacktraces as well. Primarily look for logs on the CouchDB instance where the replication jobs runs, but there may be a chance it's on one of the endpoint instances as well.

Also 3.0.0 seems a bit old and we already released 3.2.0. Another option is to try to upgrade and see if the error still persists.
